lib/lang/latin/latin-language-dataset.js

```diff
@@ -510,11 +510,11 @@ export default class LatinLanguageDataset extends LanguageDataset {
    * @return {Lemma[] | []} Array of matching Lemma objects or an empty array if nothing is found.
    */
   getMatchingIrregularLemmas (inflections) {
-    let lemmas = []
+    const lemmas = []
```

Comment:
@kirlat, it is not quite clear for me - how this eslint rule works -
`lemmas` is an array that would be changed in the following lines.
Why should it be `const`?

Reply:
Since Array is an object, it is stored by reference, and `const` guarantees only that the reference to the array will never change (i.e. that `lemmas` will always reference the same array). It does not protect array content from being changed, so we are free to add or remove element from it.